src/content/docs/ko/reference/errors/env-unsupported-get-secret.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Unsupported astro:env getSecret
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **EnvUnsupportedGetSecret**: `astro:env/server` exported function `getSecret` is not supported by your adapter.
+
+## 무엇이 잘못되었나요?
+The `astro:env/server` exported function `getSecret()` is not supported by your adapter.
```

Comment:
Looks like you forgot to delete the original here!

Reply:
Thanks for the good point.